Gives Vox Raider a dedicated map

juxjux9930 opened the pull request:
🆑 Jux
rscadd: Gives Vox Raider a dedicated map.
/🆑 

Non user-facing change is also moving the raider gun spawners to outside raider files because I thought I was going to use
 them and then didn't. 
![voxinterceptor](https://user-images.githubusercontent.com/68120725/147377032-c4c97e88-9e4a-4d1e-ae53-c262aeafc605.PNG)

SierraKomodo commented on maps/antag_spawn/vox/voxraider.dm:
```diff
@@ -0,0 +1,89 @@
+	name = "Empty Space"
+	desc = "Just some empty space, with an irregular sensor echo."
+	in_space = 1
+	known = 0
```
```suggestion
	in_space = TRUE
	known = FALSE
```

juxtaposed9930 commented:
The airlock works, and while the map needs more work, it's functional enough where I say it _could_ be merged in the current state

juxtaposed9930 commented:
Leaving as-is for feedback and review.

SierraKomodo reviewed (dismissed):
- Radio headsets only have common channel and lack a tcomms server or shortwave to transmit through.
- The airlock's canister doesn't have enough gas for the airlock to cycle back in fully after cycling out. (It reaches 100 kPa, just shy of the 101 required for it to finish the cycle).
- There are multiple engine nozzles mapped onto the same tiles, causing the engine control console to display a bunch of 0 thrust engines. (I'm amazed this didn't break anything else)
- Vox ship APC completely loses all power when it launches into open space - Probably needs a better power cell in the APC.

juxtaposed9930 commented:
> * Radio headsets only have common channel and lack a tcomms server or shortwave to transmit through.
> * The airlock's canister doesn't have enough gas for the airlock to cycle back in fully after cycling out. (It reaches 100 kPa, just shy of the 101 required for it to finish the cycle).
> * There are multiple engine nozzles mapped onto the same tiles, causing the engine control console to display a bunch of 0 thrust engines. (I'm amazed this didn't break anything else)
> * Vox ship APC completely loses all power when it launches into open space - Probably needs a better power cell in the APC.

Right, so, things that I've done. The radio situation is solved, Vox Raiders now have their own radio channel and horrid little allinone to make it work. The canister has nitrogen in it now. The strange engine overlap that shouldn't exist is now gone, and the APC is now of the hyper variety.

To get the radio thing to exist in a way that didn't hurt my small brain, I had to change how vox raiders got equipped. I hate myself for everything I made in that bit.

SierraKomodo commented on code/game/antagonist/outsider/vox.dm:
```diff
@@ -81,19 +76,16 @@ GLOBAL_DATUM_INIT(vox_raiders, /datum/antagonist/vox, new)
\ No newline at end of file
+/datum/antagonist/vox/equip(var/mob/living/carbon/human/vox/player)
+	if(!..())
+		return 0
```
```suggestion
		return FALSE
```

SierraKomodo commented on code/game/antagonist/outsider/vox.dm:
```diff
@@ -81,19 +76,16 @@ GLOBAL_DATUM_INIT(vox_raiders, /datum/antagonist/vox, new)
+	vox_raider.equip(player)
+
+
+	return 1
```
```suggestion
	return TRUE
```

SierraKomodo commented on code/game/antagonist/outsider/vox.dm:
```diff
@@ -81,19 +76,16 @@ GLOBAL_DATUM_INIT(vox_raiders, /datum/antagonist/vox, new)
-	vox.equip_to_slot_or_del(new /obj/item/clothing/gloves/vox(vox), slot_gloves)
-	vox.equip_to_slot_or_del(new /obj/item/storage/belt/utility/full(vox), slot_belt)
\ No newline at end of file
+/datum/antagonist/vox/equip(var/mob/living/carbon/human/vox/player)
```
```suggestion
/datum/antagonist/vox/equip(mob/living/carbon/human/vox/player)
```

SierraKomodo commented on code/game/antagonist/outsider/vox.dm:
```diff
@@ -16,19 +16,14 @@ GLOBAL_DATUM_INIT(vox_raiders, /datum/antagonist/vox, new)
-				var/datum/job/submap/job = submap.jobs[A]
-				job.total_positions = 0
-			return
+			return FALSE
+	return TRUE
```
Was swapping the return values here intentional?

juxjux9930 replied:
Shoulllld be switching it so that instead of requiring vox-scav to spawn, it it requires vox-scav to NOT spawn.

SierraKomodo commented on code/game/objects/items/devices/radio/encryptionkey.dm:
```diff
@@ -42,6 +42,12 @@
+	icon_state = "cypherkey"
+	channels = list("Vox Raider" = 1, "Hailing" = 1)
+	origin_tech = list(TECH_ESOTERIC = 2)
+	syndie = 1
```
```suggestion
	syndie = TRUE
```

SierraKomodo commented on code/game/objects/items/devices/radio/encryptionkey.dm:
```diff
@@ -42,6 +42,12 @@
 
+/obj/item/device/encryptionkey/vox_raider
+	icon_state = "cypherkey"
+	channels = list("Vox Raider" = 1, "Hailing" = 1)
```
```suggestion
	channels = list("Vox Raider" = TRUE, "Hailing" = TRUE)
```

juxjux9930 committed:
Gives Vox Raider a dedicated map

Mucker and Sierra my beloved
Update code/game/antagonist/outsider/vox.dm

Co-authored-by: Sierra Brown <sierrakomodo@hey.com>
Update code/game/antagonist/outsider/vox.dm

Co-authored-by: Sierra Brown <sierrakomodo@hey.com>
Update code/game/antagonist/outsider/vox.dm

Co-authored-by: Sierra Brown <sierrakomodo@hey.com>

SierraKomodo reviewed (changes requested):
All of the pipes on the N2 tank/tcomms room are broken and don't actually link to the tanks:
![dreamseeker_2021-12-27_10-38-29](https://user-images.githubusercontent.com/11140088/147498773-3b878064-f001-47df-a2d4-15e895e184c9.png)

juxtaposed9930 commented:
> All of the pipes on the N2 tank/tcomms room are broken and don't actually link to the tanks: ![dreamseeker_2021-12-27_10-38-29](https://user-images.githubusercontent.com/11140088/147498773-3b878064-f001-47df-a2d4-15e895e184c9.png)

Hrrk. Can't reproduce but I'll try to fix anyways.
